Routing/Loader/Reader/RestActionReader.php

```diff
@@ -352,16 +352,16 @@ private function getCondition(\ReflectionMethod $method, RouteAnnotation $annota
         $condition = $annotation->getCondition();
 
         if (!empty($this->versions)) {
-            $versionCondition = "request.attributes.get('version') == (";
+            $versionCondition = "request.attributes.get('version') in [";
             $first = true;
             foreach ($this->versions as $version) {
```

Comment:
Sorry there were one set of parenthesis which came from symfony language auto generation.
The real result is this:

```
request.attributes.get('version') == (<version> or <version> or <version>)
```

which then auto-generates to:

```
request.attributes.get('version') == ((<version> || <version>) || <version>)
```
like I added to the related issue #1436 

So with my fix we get:
```
request.attributes.get('version') in [<version>, <version>, <version>]
```
which auto-generates to:
```
in_array(request.attributes.get('version'), array(0 => <version>, 1 => <version>, 2 => <version>))
```

Sorry for the mix, my bad :x